src/read_write/node_writer.rs

```diff
@@ -197,7 +215,74 @@ impl WriteLEPos for AttributeData {
     }
 }
 
+pub trait WriteLEEncoded {
```

Comment:
NIt: Could this go into codec.rs or would that not make sense in the current dependency graph? Also, how about naming this `WriteEncoded` since future encodings (like compression) might not really be LE anymore?

Reply:
I thought about it, but since it takes the `DataWriter` as parameter, I decided to leave it here to be consistent with the other two traits... I will rename it.